urllib3/response.py

```diff
@@ -387,48 +386,67 @@ def readinto(self, b):
             b[:len(temp)] = temp
             return len(temp)
 
-    def read_chunked(self, amt=None):
+    def _get_next_chunk(self):
+        # First, we'll figure out length of a chunk and then
+        # we'll try to read it from socket.
+        if self.chunk_left is None:
+            line = self._fp.fp.readline()
+            line = line.decode()
```

Comment:
So I expect this is necessary on Python 3 because Python 3 doesn't have the same methods as strings do. I have an idea to avoid this though. I'll push it up shortly